plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -230,6 +297,40 @@ class Bazel(
             )
         }
 
+        packageIdsWithArchiveOverride.mapTo(result) {
+            val archiveOverride = archiveOverrides.getValue(it.name)
+
+            if (archiveOverride.urLs.size > 1) {
+                logger.warn {
+                    "The module ${it.name} has multiple URLs ${archiveOverride.urLs} defined in the archive override," +
+                        "but ORT only supports one URL for source artefacts. Only the first URL will be used."
+                }
+            }
+
+            val hash = archiveOverride.integrity?.let { integrity ->
+                integrity.split('-', limit = 2).let { (algo, b64digest) ->
```

Comment:
Prefer a destructing declaration to reduce nesting:

    val (algo, b64digest) = integrity.split('-', limit = 2)
    // ...